### ISSUE TITLE
on Apply Text Spacing the code is getting truncated

### ISSUE THREAD

**Issue by fsteffi** [microsoft/vscode#171701]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: Version: 1.75.0-insider
- OS Version: 25276.1000

## Steps to Reproduce:

1. open the environment:https://insiders.vscode.dev/
2. Add the line of code"/* Game of Life * Implemented in TypeScript * To learn more about TypeScript, please visit http://www.typescriptlang.org/please visit http://www.typescriptlang.org/please visit http://www.typescriptlang.org/ */ test all this inline data"   in the editor
3. Run the text spacing bookmark

## Actual:
on running the text spacing bookmark on the code "/* Game of Life * Implemented in TypeScript * To learn more about TypeScript, please visit http://www.typescriptlang.org/please visit http://www.typescriptlang.org/please visit http://www.typescriptlang.org/ */ hello world type"

the last words in the code are truncated and even on scrolling to it cant be seen.

the visual text is "/* Game of Life * Implemented in TypeScript * To learn more about TypeScript, please visit http://www.typescriptlang.org/please visit http://www.typescriptlang.org/please visit http://"

## Expected:
once text spacing is applied the full text should be visible as seen without text spacing 

## User Impact:
Keyboard user or low vision users will not get to full the full string of text

## Attachment
[on Apply Text Spacing the code is getting truncated.webm](https://user-images.githubusercontent.com/24580570/213455217-4c4c3092-92ba-4777-ae40-2e2d42d14e92.webm)

**GitHub Tags**:#A11yUsable;#A11yTCS;#A11ySev4;#DesktopWeb;#Win11;#Visual Studio Code Client;#a11y-monacoeditor;#TextSpacing;#Chromiumedge;

**Comment by isidorn** [comment]
What is the "3. Run the text spacing bookmark"?

Assigning to @hediet

**Comment by ailsaleen** [comment]
@fsteffi may be able to clarify, as they filed the original bug. 
This issue also occurs in AzureML, and we were given the following repro steps for AzureML: 

Step 1: Get the “Text spacing” extension from google: https://chrome.google.com/webstore/detail/stylus/clngdbkpkpeebahjckkjfobafhncgmne 

Step 2: Click “Manage” control in the text spacing tool.

Step 4: Select “Add new style” button.

Step 3: Add below code.

* { 

     line-height: 1.5 !important; 

     letter-spacing: 0.12em !important; 

     word-spacing: 0.16em !important; 

} 

p { 

     margin-bottom: 2em !important; 

}

**Comment by fsteffi** [comment]
hi @isidorn  the steps 3 is a code to run text spacing 
https://codepen.io/stevef/pen/YLMqbo/

**Comment by hediet** [comment]
Unfortunately, we cannot support injected CSS/JS scripts, as they break our assumptions about how characters are layouted.

Please create a feature-request/issue directly, so we can try to prioritize and implement what the injected script tries to do.

Thank you!

**Comment by scottaohara** [comment]
Speaking out this internally to @isidorn, there are settings provided which can be adjusted by the user to meet their individual preferences and achieve the goal of the [text spacing criterion](https://www.w3.org/WAI/WCAG21/Understanding/text-spacing.html)

These settings, however, were initially missed when reviewing the settings under "font", as that was where they were expected to be located, but they were instead located somewhere else in the UI.  Ideally, if not moved to the font setting, they would at least be called out (linked to) in the font setting screen to ensure their existence was not missed.

**Comment by isidorn** [comment]
Thanks @scottaohara 
For transparency these settings are `editor.letterSpacing` and `editor.lineHeight`

We should add a word "font" to the description of these settings if possible.

We got confirmation that this is a Sev3 issue, which means that we do not have to fix it. However let me reopen and assign to backlog for now.

**Comment by fsteffi** [comment]
bug has been updated to a usable bug as per team conversation with Scott "Discuss on issue about Audio cues in VS Studio and VS Code
"

**Comment by fsteffi** [comment]
github tags: #A11yMAS;#a11y-partner;

**Comment by isidorn** [comment]
This is sev3. We have decided not to fix this. Thus closing as out of scope.

**Comment by fsteffi** [comment]
#closed;
